Kickstarter-iOS/Features/PaymentMethods/Controller/PaymentMethodsViewController.swift

```diff
@@ -143,6 +153,66 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
 
   // MARK: - Private Helpers
 
+  private func goToPaymentSheet(data: PaymentSheetSetupData) {
+    PaymentSheet.FlowController
+      .create(
+        setupIntentClientSecret: data.clientSecret,
+        configuration: data.configuration
+      ) { [weak self] result in
+        guard let strongSelf = self else { return }
+
+        switch result {
+        case let .failure(error):
+          // strongSelf.viewModel.inputs.shouldCancelPaymentSheetAppearance(state: true)
+          strongSelf.messageBannerViewController?
+            .showBanner(with: .error, message: error.localizedDescription)
+        case let .success(paymentSheetFlowController):
+          strongSelf.paymentSheetFlowController = paymentSheetFlowController
+          strongSelf.paymentSheetFlowController?.presentPaymentOptions(from: strongSelf) { [weak self] in
+            guard let strongSelf = self else { return }
+
+            /** Handle updating payment methods with new card/errors
```

Comment:
Is this going to be implemented in another ticket? If so, would it be helpful to mention which one in the comment?